photon-client/src/main/scala/com/linkedin/photon/ml/cli/game/training/Driver.scala

```diff
@@ -248,15 +250,16 @@ final class Driver(val sc: SparkContext, val params: GameParams, implicit val lo
    * Write the GAME models to HDFS.
    *
    * TODO: Deprecate model-spec then remove it in favor of model-metadata, but there are clients!
+   * TODO: Should we perform model selection for NONE output mode? NONE output mode is used mostly as a debugging tool.
    *
    * @param featureShardIdToFeatureMapLoader The shard ids
    * @param models All the models that were producing during training
    * @param bestModel The best model
    */
   protected[training] def saveModelToHDFS(
-      featureShardIdToFeatureMapLoader: Map[String, IndexMapLoader],
-      models: Seq[(GAMEModel, Option[EvaluationResults], String)],
-      bestModel: Option[(GAMEModel, EvaluationResults, String)]): Unit =
+    featureShardIdToFeatureMapLoader: Map[String, IndexMapLoader],
+    models: Seq[(GAMEModel, Option[EvaluationResults], String)],
+    bestModel: Option[(GAMEModel, EvaluationResults, String)]): Unit =
```

Comment:
Is this one settable in Scala code style in IDEA? In any case, I fixed the file. Thanks!